src/app/copy/inline-copy/inline-copy.component.html

```diff
@@ -0,0 +1,16 @@
+<span
+  class="pfng-inline-copy">
+  <span
+    class="pfng-inline-copy-txt-cont"
+    data-toggle="tooltip"
+    data-placement="top"
```

Comment:
I didn't realize those where old, good catch! I was a little hesitant to use placement/tooltip at first because they are not valid html5 attributes and are reported as error in tools like markup validation service - https://validator.w3.org/, but to be fair it's really just a conformance rule that everyone breaks anyway. I'll go with the ngx-bootstrap attributes for now.